### DISCUSSION TITLE
Did i buy the right thing?

### DISCUSSION THREAD

**Discussion by karlfriendnz** [primefaces/community#2722] [Figma UI Kit]
Hi there I have just purchased the (https://primeblocks.org/pricing) I wanted the prime vue UI kit - have I purchased the right thing - when i downloaded the kit it looks like it was just for prime blocks which is not what I want - I would like to have access to a figma file that has all the components of Prime Vue - please help 

Should I have purchased this instead - https://www.primefaces.org/store/uikit.xhtml

Are they the same thing?

**Comment by onursenture** [comment]
Yes, it seems that you have made an incorrect purchase. You can write to contact -at- primetek.com.tr regarding the issue.
